index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/1ca239d8ab.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version Portfolio</title>
+</head>
+<body>
+  <div class="container">
+
+    <section class="container-section">
+    <header>
+      <nav class="nav-header">
+        <div class="logo">My Logo</div>
+        <span class="bars"><i class="fa-solid fa-bars"></i></span>
+      </nav>
+    </header>
+    
+      <div class="intro">
+        <h1>I'm Dinma <br> Glad to see <br> you!</h1>
```

Comment:
- [ ] It would be great if you could apply the right font-weight to your `h1` element so as to follow the template design
- Your site:
![Annotation 2022-03-15 172226](https://user-images.githubusercontent.com/85113567/158424902-32e14114-38ff-4f34-88d3-6193a5a7618e.jpg)
- Figma design:
![Annotation 2022-03-15 172211](https://user-images.githubusercontent.com/85113567/158424927-65c6bb70-5738-4e84-ba81-81c02e5c4b58.jpg)